src/babel-preset-adeira/README.md

```diff
@@ -240,6 +241,25 @@ Using plugins:
 Using polyfills: No polyfills were added, since the `useBuiltIns` option was not set.
 ```
 
+## New jsx transform
+
+This packages supports the new jsx transform from `1.2.0`. You can use it like this:
```

Comment:
I'd include some link here as well. https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html would be a good adept